examples/src/routes/VisualizationByIdentifier.jsx

```diff
@@ -144,6 +146,12 @@ export const VisualizationByIdentifier = () => (
             for={VisualizationMeasureValueFilterByIdentifierExample}
             source={VisualizationMeasureValueFilterExampleSRC}
         />
+
+        <h2 id="measure-value-filter-treat-null-as-zero">Measure Value Filter Treat Null As Zero</h2>
```

Comment:
Maybe change it to "Measure Value Filter that treats measure null values as zero"